user/conf/database-example-for-docker.conf

```diff
@@ -0,0 +1,8 @@
+// Melia
```

Comment:
Do we need an example db conf file for Docker? Will this not be created during the setup process?